public/data/marisabrantley.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "Marisa Brantley",
+  "type": "personal",
+  "bio": "Former paralegal studying to become a front-end dev",
+  "avatar": "https://github.com/marisabrantley.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/marisabrantley",
+      "icon": "github"
+    },
+    {
+      "name": "Let's connect on Twitter",
+      "url": "https://twitter.com/marisabrantley",
+      "icon": "twitter"
+    },
+    {
+      "name": "Join me on LinkedIn",
+      "url": "https://www.linkedin.com/in/marisabrantley/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "See my pics on Instagram",
+      "url": "https://www.instagram.com/marisab_oc/",
+      "icon": "instagram"
+    }
+  ]
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```